modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -300,11 +306,28 @@ void ONNXImporter::addLayer(Net& dstNet, LayerParams& layerParams,
     }
 }
 
+void addConstantNodesForInitializers(opencv_onnx::GraphProto& graph_proto)
+{
+    int num_initializers = graph_proto.initializer_size();
+    for (int id = 0; id < num_initializers; id++)
+    {
+        opencv_onnx::TensorProto initializer = graph_proto.initializer(id);
+        opencv_onnx::NodeProto* constant_node = graph_proto.add_node();
+        constant_node->set_op_type("Constant");
+        constant_node->set_name(initializer.name());
+        constant_node->add_output(initializer.name());
+        opencv_onnx::AttributeProto* value = constant_node->add_attribute();
+        opencv_onnx::TensorProto* tensor = value->add_tensors();
```

Comment:
Maybe let's copy it to `t` field instead of `tensors`?

Reply:
@dkurt t I was trying to do so but couldn't find any function for that so I had to copy it to `tensor`. Maybe I missed it. I am checking it, please tell me if you have that in your mind.

Reply:
There is `set_allocated_t` in https://github.com/opencv/opencv/blob/master/modules/dnn/misc/onnx/opencv-onnx.pb.h

Reply:
Done! thanks